x/concentrated-liquidity/lp.go

```diff
@@ -16,8 +16,11 @@ import (
 // to create position if the desired amounts cannot be satisfied.
 // On success, returns an actual amount of each token used and liquidity created.
 // Returns error if:
-// TODO: list error cases
-// TODO: table-driven tests
+// - the provided ticks are out of range / invalid
+// - the pool provided does not exist
+// - the liquidity delta is zero
+// - the amount0 or amount1 returned from the position update is less than the defined minimums
```

Comment:
I got confused thinking that we defined them in code but these are given as input

```suggestion
// - the amount0 or amount1 returned from the position update is less than the given minimums
```